substrate/frame/system/src/tests.rs

```diff
@@ -721,7 +721,7 @@ fn extrinsics_root_is_calculated_correctly() {
 		System::note_finished_extrinsics();
 		let header = System::finalize();
 
-		let ext_root = extrinsics_data_root::<BlakeTwo256>(vec![vec![1], vec![2]]);
+		let ext_root = extrinsics_data_root::<BlakeTwo256>(vec![vec![1], vec![2]], sp_core::storage::StateVersion);
```

Comment:
I don't understand how this works, shouldn't you provide a concrete enum variant instead of type here? Looks like it shouldn't compile at all.